Core/cptCommonFieldMap_bas.bas

```diff
@@ -0,0 +1,154 @@
+Attribute VB_Name = "cptCommonFieldMap_bas"
+'<cpt_version>v1.0.1</cpt_version>
+Option Explicit
+
+#If VBA7 Then
+
+    Private Declare PtrSafe Function GetPrivateProfileStringA Lib "Kernel32" (ByVal strSection As String, _
+        ByVal strKey As String, ByVal strDefault As String, _
+        ByVal strReturnedString As String, _
+        ByVal lngSize As Long, ByVal strFileNameName As String) As Long
+        
+    Private Declare PtrSafe Function WritePrivateProfileStringA Lib _
+        "Kernel32" (ByVal strSection As String, _
+        ByVal strKey As String, ByVal strString As String, _
+        ByVal strFileNameName As String) As Long
+
+#Else
+
+    Private Declare Function GetPrivateProfileStringA Lib "Kernel32" (ByVal strSection As String, _
+        ByVal strKey As String, ByVal strDefault As String, _
+        ByVal strReturnedString As String, _
+        ByVal lngSize As Long, ByVal strFileNameName As String) As Long
+        
+    Private Declare Function WritePrivateProfileStringA Lib _
+        "Kernel32" (ByVal strSection As String, _
+        ByVal strKey As String, ByVal strString As String, _
+        ByVal strFileNameName As String) As Long
+    
+#End If
+
+Function cptGetCustomFieldName(ByVal FieldName As String) As String
+
+    Dim settingsFile As String
+    
+    settingsFile = GetSettingsFile
+
+    cptGetCustomFieldName = GetPrivateProfileString(settingsFile, FieldName, "Name")
+    
+End Function
+
+Function cptGetCustomFieldGUID(ByVal FieldName As String) As String
+
+    Dim settingsFile As String
+    
+    settingsFile = GetSettingsFile
+
+    cptGetCustomFieldName = GetPrivateProfileString(settingsFile, FieldName, "GUID")

```

Comment:
I think this should be `cptGetCustomFieldGUID = GetPrivateProfileString(settingsFile, FieldName, "GUID")`

Reply:
corrected in branch